tests/unit/setup-context-test.js

```diff
@@ -0,0 +1,110 @@
+import { module, test } from 'qunit';
+import Service from '@ember/service';
+import { setupContext } from 'ember-test-helpers';
+import hasEmberVersion from 'ember-test-helpers/has-ember-version';
+import { setResolverRegistry, createCustomResolver } from '../helpers/resolver';
+
+module('setupContext', function(hooks) {
+  hooks.before(function() {
+    setResolverRegistry({
+      'service:foo': Service.extend({ isFoo: true }),
+    });
+  });
+
+  module('without options', function(hooks) {
+    let context;
+    hooks.beforeEach(function() {
+      context = {};
+      setupContext(context);
+    });
+
+    test('it sets up this.owner', function(assert) {
+      let { owner } = context;
+      assert.ok(owner, 'owner was setup');
+      assert.equal(
+        typeof owner.lookup,
+        'function',
+        'has expected lookup interface'
+      );
+
+      if (hasEmberVersion(2, 12)) {
+        assert.equal(
+          typeof owner.factoryFor,
+          'function',
+          'has expected lookup interface'
+        );
+      }
+    });
+
+    test('it uses the default resolver if no override specified', function(
+      assert
+    ) {
```

Comment:
prettier doing some strange formatting here? should we increase the max line length to something like 100 or 120?